packages/governance/src/constants.js

```diff
@@ -14,6 +14,9 @@ export const ParamTypes = /** @type {const} */ ({
   NAT: 'nat',
   RATIO: 'ratio',
   STRING: 'string',
+
+  // passable record
```

Comment:
WDYT of disambiguating in the key?

Reply:
Probably a good idea. I resisted because of the length, but that's probably not an important consideration.